py/kubeflow/testing/notebook_tests/execute_notebook.py

```diff
@@ -62,6 +64,15 @@ def run_notebook_test(notebook_path, parameters=None):
   html_exporter = nbconvert.HTMLExporter()
   (html_output, _) = html_exporter.from_notebook_node(nb)
   gcs_path = os.getenv("OUTPUT_GCS")
+
+  # Per https://github.com/kubeflow/testing/issues/715
+  # we need to add some uniquness to the name since different test runs
+  # will use the same OUTPUT_GCS directory
+  subdir = datetime.datetime.now().strftime("%Y%m%d-%H%M")
+  subdir = "-" + uuid.uuid4().hex[0:4]
```

Comment:
Do we wish to combine the date and the uuid? The date is just overwritten.

Reply:
@NikeNano Good catch thanks; it was a bug.

Reply:
Fixed. @NikeNano PTAL